datalad_catalog/catalog.py

```diff
@@ -366,25 +371,23 @@ def _create_catalog(
     msg = ""
     if not catalog.is_created():
         catalog.create()
-        msg = f"Catalog successfully created at: {catalog.location}"
+        msg = ("Catalog successfully created at: %s", catalog.location)
     else:
         if force:
             catalog.create(force)
-            msg = f"Catalog assets successfully overwritten at: {catalog.location}"
+            msg = (
+                "Catalog assets successfully overwritten at: %s",
+                catalog.location,
+            )
     # Yield created/overwitten status message
-    yield get_status_dict(
-        action="catalog create", path=abspath(curdir), status="ok", message=msg
-    )
+    yield get_status_dict(**res_kwargs, status="ok", message=msg)
     # If metadata was also supplied, add this to the catalog
     if metadata is not None:
-        yield from _add_to_catalog(
-            catalog, metadata, dataset_id, dataset_version, force, config_file
-        )
+        yield from _add_to_catalog(catalog, metadata, res_kwargs)
```

Comment:
One thing I'd like to point out.

This particular change complements the refactoring in #127 and makes it possible (again) to run `datalad create -c my-cat -m my-meta.jsonl` (so "fixes a regression" technically speaking).

If this PR needs to wait for some reason, then it could be worth introducing this change alone (sans `res_kwargs` then).

Reply:
good to know, thx!